MachObfuscator/Mach/Mach+Loading.swift

```diff
@@ -191,7 +191,8 @@ private extension Mach.Platform {
             self = .macos
         case UInt32(PLATFORM_IOSSIMULATOR):
             self = .ios
-        case UInt32(PLATFORM_IOSMAC):
+        case 6: // `PLATFORM_IOSMAC` renamed to `PLATFORM_MACCATALYST` in Xcode 11.0.
```

Comment:
maybe `#if compiler(>=5.1)` would to the trick and would be more readable because you could use named constants?

Reply:
I didn't know that one :D Thanks!